app/controllers/shelters_controller.rb

```diff
@@ -4,6 +4,10 @@ def index
       @shelters = Shelter.order_by_number_of_pets
     elsif params[:search].present?
       @shelters = Shelter.search(params[:search])
+    elsif params[:admin] == "admin"
+      @admin = true
+      @shelters = Shelter.order_by_sql
+      @shelters_with_applications = Shelter.pending_applications
```

Comment:
We don't want to create more things in the controller then we need. Remember we want our controllers to be as slim as possible. Line 10 can go away and you can just call `.pending_applications` on your instance `@shelters` in your view.